tools/toolchains/arm.py

```diff
@@ -31,12 +31,18 @@
 
 ARMC5_MIGRATION_WARNING = (
     "Warning: We noticed that you are using Arm Compiler 5. "
-    "We are deprecating the use of Arm Compiler 5 soon. "
+    "We are deprecating the use of Arm Compiler 5. "
     "Please upgrade your environment to Arm Compiler 6 "
     "which is free to use with Mbed OS. For more information, "
     "please visit https://os.mbed.com/docs/mbed-os/latest/tools/index.html"
 )
 
+UARM_TOOLCHAIN_WARNING = (
+    "Warning: We noticed that you are using uARM Toolchain. "
+    "We are deprecating the uARM Toolchain. "
```

Comment:
```suggestion
    "We are deprecating the use uARM Toolchain. "
```
To match form of `ARMC5_MIGRATION_WARNING`.

Reply:
I prefer to describe like "We are deprecating the use of uARM Toolchain. "

Reply:
It is not about preferences but rather consistency.

Reply:
i am not getting what is the meaning of consistency like just following the previous ARMC5_MIGRATION_WARNING. i would rather keep like this "We are deprecating the uARM Toolchain."  which is not wrong and not confusing the user.

Reply:
`ARMC5_MIGRATION_WARNING` does not say `We are deprecating the Arm Compiler 5` it says `We are deprecating the use of Arm Compiler 5`. Consistency in the way the sentence is formed.